### DISCUSSION TITLE
Can't open map file in Purple Pen

### DISCUSSION THREAD

**Discussion by IamShado** [OpenOrienteering/mapper#2000] [General]
I'm getting an error when trying to open an omap file in Purple Pen.  It worked fine before but at some point after working on the file I started getting this error: "Cannot read map for the following reason: 'Input string was not in a correct format.' Opening and re-saving the map in the latest version of OpenOrienteering Mapper may fix problems."

I've tried re-saving.  Didn't fix.   I have the latest version of OOM.

I tried deleting the entire contents of the map, deleting all symbols and deleting all colors and saving... still getting the error.

I tried resetting georeferencing.. nothing.

I deleted all templates and map parts.   It's basically just a white map.  Nothing.

Seems something got corrupted in the file somewhere and I don't know how I can fix it other than creating a completely new map and trying to copy everything over.   

I can, however, save it as an ocad map and it opens just fine.  

Any suggestions?

**Comment by IamShado** [comment]
Well, I don't know why I didn't think of this before but I saved the file as an ocad file then opened that up and re-saved as an oom file then it fixed the error.  Still curious what happened but I guess this works.

**Reply by pkturner** [reply]
When you delete all the map objects, Mapper keeps them all around in case you change your mind and want to undo. To really clear out the saved map you need to "Clear undo / redo history", found in the Edit menu.
